packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -72,12 +79,15 @@ export class VaTelephone {
         formattedNum = `${area}-${local}-${vanity} (${last4})`;
       }
     }
+    if (tty) {
+      formattedNum = `TTY: ${formattedNum}`;
```

Comment:
I didn't see any guidance that a TTY number _couldn't_ be a 10 digit number, so that's why this isn't inside an else block.

Reply:
Would you mind checking with Sarah to verify this?

Reply:
@SarahKay8 do you know if TTY numbers are always 3 digits or if it's possible for them to be 10 digits?

Reply:
It looks like 10 digit TTY numbers [are possible](https://github.com/department-of-veterans-affairs/vets-website/blob/5a07edbdeb4987b90716bce44d0b4256ee4b4c9b/src/applications/static-pages/health-care-manage-benefits/secure-messaging-page/components/UnauthContent/index.js#L163-L165).